src/cli_validator/cmd_meta/parser.py

```diff
@@ -139,8 +138,6 @@ def load_meta(self, meta):
                 'default': param.get('default'),
                 'type': self.convert_type(param.get('type')) if param.get('type') else None,
             }
-            if 'required' in param and len(param['options']) > 0:
-                kwargs['required'] = param['required']
             self.add_argument(*param['options'], **kwargs)
```

Comment:
```suggestion
            self.add_argument(*param['options'], **kwargs)
        self.add_argument('--ids', dest='ids')
```
You should add a new argument in parser to support parsing `--ids`

Reply:
Yes, I forgot to add it...